packages/permissions/src/utils/has-permissions.spec.tsx

```diff
@@ -205,3 +206,178 @@ describe('getInvalidPermissions', () => {
     });
   });
 });
+
+describe('hasAppliedDataFence', () => {
+  describe('user has not datafence permissions', () => {
+    it('should return false', () => {
```

Comment:
Nit: I wouldn't indicate the return value but what it means: so here "should indicate not having applied datafences".